src/recon_buildblock/PoissonLogLikelihoodWithLinearModelForMeanAndProjData.cxx

```diff
@@ -512,7 +512,7 @@ actual_subsets_are_approximately_balanced(std::string& warning_message) const
 			 view_num <= this->proj_data_sptr->get_max_view_num();
 			 view_num += this->num_subsets)
 		  {
-			const ViewSegmentNumbers view_segment_num(view_num, segment_num);
+			const ViewSegmentTOFNumbers view_segment_num(view_num, segment_num);
```

Comment:
probably needs TOF arg

Reply:
And TOF loop in subset sensitivity computation?

Reply:
again, normally no TOF loop in sensitivity (as all TOF kernels sum to 1).